packages/components/psammead-media-player/README.md

```diff
@@ -19,18 +19,33 @@ This component to be used at any point on the page, specifically when a media pl
 | Argument  | Type                | Required | Default | Example         |
 |-----------|---------------------|----------|---------|-----------------|
 | `src` | string | Yes   | - | `http://foobar.com/embeddable_endpoint` |
+| `skin` | string | No   | `"classic"` | `"audio"` |
 | `title` | string | Yes | - | `Video player` |
+| `service` | string | Yes   | - | `"news"` |
 | `showPlaceholder` | boolean | No   | `true` | `false` |
 | `placeholderSrc` | string | No   | `null` | `http://foobar.com/placeholder.png` |
-| `placeholderSrcset` | string | No   | - | `"https://bbc.com/300/cat.jpg 300w, https://bbc.com/450/cat.jpg 450w, https://bbc.com/600/cat.jpg 600w"` |
+| `placeholderSrcset` | string | No   | `null` | `"https://bbc.com/300/cat.jpg 300w, https://bbc.com/450/cat.jpg 450w, https://bbc.com/600/cat.jpg 600w"` |
 | `portrait` | boolean | No   | `false` | `true` |
+| `[mediaInfo](#mediaInfo)` | object | Yes   | - | `{ title: 'A vertical video pretending to be a cat title',duration: '0:11',durationSpoken: '2 minutes 11 seconds',datetime: 'PT11S',type: 'video' }`|
 
 The `src` prop is required, as it tells the component what page it needs to embed.
 The `placeholderSrcset` prop is not required, as it allows image responsiveness and optimization depending on the size of the screen.
 The `title` prop is required for accessibility of the embedded iframe.
 The `portrait` prop is not required, and defaults to `false`. This is to support portrait video content in the future.
 The `showPlaceholder` boolean prop is also not required, and defaults to `true`.
 Assuming `showPlaceholder` is `true`, the `placeholderSrc` will be what image to display as the placeholder.
+The `mediaInfo` prop is required, and has the following properties.
+
+#### mediaInfo
+
+| Argument  | Type                | Required | Default | Example         |
+|-----------|---------------------|----------|---------|-----------------|
+| `title` | string | Yes   | - | `"A vertical video pretending to be a cat title"` |
+| `datetime` | string | No| - | `"PT11S"` |
+| `duration` | string| No   | - | `"0:11"` |
+| `durationSpoken` | string | No   | - | `"2 minutes 11 seconds"` |
+| `type` | string | No   | `"video"` | `"video"` |
```

Comment:
Would it make sense to show `"audio"` as the example here, so readers know it's an available option?

Reply:
alright, on it

Reply:
done

Reply:
done